core/templates/components/state-directives/outcome-editor/outcome-editor.component.spec.ts

```diff
@@ -406,6 +406,23 @@ describe('Outcome Editor Component', () => {
     expect(component.isSelfLoopDestStuck(outcome)).toBeFalse();
   });
 
+  it('should check if a destination for stuck learner is null', () => {
+    let outcome = new Outcome(
+      'Me Llamo',
+      null,
+      new SubtitledHtml('<p> Previous HTML string </p>', 'Id'),
+      true,
+      [],
+      null,
+      null,
+    );
+
+    spyOn(stateEditorService, 'getActiveStateName')
+      .and.returnValue('Introduction');
+
+    expect(component.isSelfLoopDestStuck(outcome)).toBe(true);
```

Comment:
Should we have a precheck for this?

Reply:
I have changed and correct it .
PTAL.
thanks